ci/run_envoy_docker.sh

```diff
@@ -98,5 +98,6 @@ docker run --rm \
        -e SYSTEM_STAGEDISPLAYNAME \
        -e SYSTEM_JOBDISPLAYNAME \
        -e SYSTEM_PULLREQUEST_PULLREQUESTNUMBER \
+       -e GOPROXY=direct \
```

Comment:
Let's omit this.  It may not be applicable to  Envoy, and is a direct artifact of building in our environment

Reply:
yeah that slipped in.